akka-persistence-testkit/src/test/scala/akka/persistence/testkit/javadsl/SnapshotNOTSerializeSpec.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2019-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.testkit.javadsl
+
+import akka.actor.Props
+import akka.persistence.testkit._
+
+class SnapshotNOTSerializeSpec extends CommonSnapshotTests {
```

Comment:
Maybe spell this `SnapshotNotSerializeSpec`?

Reply:
corrected